docs/migration.md

```diff
@@ -11,30 +11,31 @@ Open your app's package.json file and replace the ACP-prefixed packages with the
 "dependencies": {
     "react-native": "0.64.2",
 -   "@adobe/react-native-acpcore": "^2.0.0"
-+   "@adobe/react-native-aepcore": "^1.0.0",
++   "@adobe/react-native-aepcore": "^2.0.0",
     ...
 },
 
 ```
 At this time, the following ACP-prefix libraries can be switched out with their respective AEP-prefix libraries.  However, ACP and AEP React Native libraries are not compatible. For extensions not supported in AEP-prefixed libraries, you should remove those packages from your package.json file. 
 
-| React Native (ACP 2.x) | React Native (AEP 1.x) |
+| React Native (ACP) | React Native (AEP) |
 | :--- | :--- |
 | @adobe/react-native-acpcore | @adobe/react-native-aepcore |
 | @adobe/react-native-acpuserprofile | @adobe/react-native-aepuserprofile |
-| @adobe/react-native-acpplaces | NA |
+| @adobe/react-native-acpplaces | @adobe/react-native-aepplaces |
 | @adobe/react-native-acpplaces-monitor | NA |
 | @adobe/react-native-acpanalytics | NA |
 | @adobe/react-native-acpmedia | NA |
 | @adobe/react-native-acpaudience | NA |
-| @adobe/react-native-acptarget | NA |
-| @adobe/react-native-acpcampaign | NA |
-| @adobe/react-native-aepassurance:2.x (compatible with ACP libraries) | @adobe/react-native-aepassurance:3.x (compatible with AEP libraries)|
+| @adobe/react-native-acptarget | @adobe/react-native-aeptarget |
+| @adobe/react-native-acpcampaign | @adobe/react-native-aepcampaignclassic |
```

Comment:
the react-native-acpcampaign wrapper is for campaign standard. the aepcampaignclassic is brand new and does not have an existing acp version. the aepcampaignstandard wrapper is to be done.